docs/index.md

```diff
@@ -0,0 +1,9 @@
+# Overview
+
+Trino Gateway is a load balancer, proxy server, and configurable routing 
+gateway for multiple [Trino](https://trino.io) clusters.
+
+<img src="./assets/logos/trino-gateway-h.png"/>
+
+
+<img src="./assets/logos/trino-gateway-v.png"/>
```

Comment:
Showing single logo is sufficient for me.

Reply:
Done